zerver/webhooks/clubhouse/view.py

```diff
@@ -676,6 +676,8 @@ def send_stream_messages_for_actions(
     "story_update_batch": get_story_update_batch_body,
 }
```

Comment:
It's a bit weird that the list of event types has a mix of underscores and dashes.  Not sure what we should do about it.